### PR TITLE
All changes

### PR DESCRIPTION
**Pull request by joseph-wortmann**

### CODE REVIEW

**Comment by joseph-wortmann** [comment]
Sorry if I violated any protocol with the pull request. Somewhat of a noob doing this. Hopefully you find the changes useful and can incorporate them in part or in whole.

Joe

**Comment by SeanRoy** [comment]
No, not at all.  Thanks for your contribution, you did everything right.  I just haven't had a chance to review your PR and thoroughly test it.  I'll probably have some questions about the issue in general once things calm down at work.

Sean

**Comment by joseph-wortmann** [comment]
Good! Happy to answer any questions you might have. I'm using these revisions locally in production already.

Joe

File: pom.xml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>com.github.seanroy</groupId>
+    <groupId>com.quinovas</groupId>
```
Comment (SeanRoy) [inline pom.xml]: You need to change this back.

File: pom.xml [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
     <artifactId>lambda-maven-plugin</artifactId>
     <packaging>maven-plugin</packaging>
     <version>2.3.2</version>
 
     <name>lambda-maven-plugin Maven Mojo</name>
     <description>A maven plugin that deploys functions to AWS Lambda</description>
-    <url>https://github.com/SeanRoy/lambda-maven-plugin</url>
+    <url>https://github.com/QuiNovas/lambda-maven-plugin</url>
```
Comment (SeanRoy) [inline pom.xml]: Change this back please

File: pom.xml [hunk 3]
```diff
@@ -19,13 +19,17 @@
 
     <developers>
         <developer>
-            <name>Sean N. Roy</name>
-            <email>Sean.Roy@gmail.com</email>
-            <organizationUrl>https://github.com/SeanRoy/lambda-maven-plugin</organizationUrl>
```
Comment (SeanRoy) [inline pom.xml]: Please change this back.

File: pom.xml [hunk 4]
```diff
@@ -19,13 +19,17 @@
...
+            <email>jwortmann@quinovas.com</email>
+            <organizationUrl>https://github.com/QuiNovas/lambda-maven-plugin</organizationUrl>
         </developer>
     </developers>
 
     <contributors>
+        <contributor>
+            <name>Sean N. Roy</name>
+            <email>Sean.Roy@gmail.com</email>
+        </contributor>
```
Comment (SeanRoy) [inline pom.xml]: Please swap us.

**Comment by joseph-wortmann** [comment]
Happy too. Didn't exactly know what to do with the POM as we were using it locally. Changes on the way in a sec.

**Comment by joseph-wortmann** [comment]
Done.

**Comment by joseph-wortmann** [comment]
Thanks Sean!

**Pull request merged**